### DISCUSSION TITLE
Idea regarding RTC on the new Mini+ (MMP)

### DISCUSSION THREAD

**Discussion by knightoftherelm** [OnionUI/Onion#701] [Ideas]
Not 100% if this has been mentioned before as I had a look on here but as the Mini+ has built in WiFi I thought it would be a good idea if there was a way for OnionOS to Auto set the Date/Time when it connects to a WiFi network.

**Comment by schmurtzm** [comment]
Done.
https://onionui.github.io/docs/apps/tweaks#set-automatically-via-the-internet
